qa/tasks/ceph.py

```diff
@@ -426,14 +426,23 @@ def cephfs_setup(ctx, config):
     # If there are any MDSs, then create a filesystem for them to use
     # Do this last because requires mon cluster to be up and running
     if mdss.remotes:
-        log.info('Setting up CephFS filesystem...')
-
-        fs = Filesystem(ctx, name='cephfs', create=True,
-                        ec_profile=config.get('cephfs_ec_profile', None))
-
-        max_mds = config.get('max_mds', 1)
-        if max_mds > 1:
-            fs.set_max_mds(max_mds)
+        log.info('Setting up CephFS filesystem(s)...')
+        cephfs_names = config.get('cephfs')
```

Comment:
Be aware @lxbsz is planning to reorganize this: https://github.com/ceph/ceph/pull/37629#discussion_r503631207

Reply:
Thanks for pointing this out. I will follow what's happening in that PR and make changes accordingly.